docs/firmware/Flashing.rst

```diff
@@ -0,0 +1,23 @@
+
+Flashing the Firmware
+#####################
+
+.. body_start
+
+First, make sure you have `Segger JLink <https://www.segger.com/downloads/jlink/>`_ installed in your path.
```

Comment:
Is path here the PATH environment variable?

Reply:
No, it refers to the current path the user is in.